ci-operator/step-registry/ipi/conf/gcp/compute-node-type/ipi-conf-gcp-compute-node-type-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+YAML_CONFIG="${SHARED_DIR}/install-config.yaml"
+
+JSON_CONFIG="${SHARED_DIR}/install-config.json"
+
+PATCH="${SHARED_DIR}/node_type.json.patch"
+
+yq-go r "${YAML_CONFIG}" -j > "${JSON_CONFIG}"
+
+jq --arg COMPUTE_NODE_TYPE "$COMPUTE_NODE_TYPE" '
+    .controlPlane.platform.gcp.osDisk.diskType = "hyperdisk-balanced" |
```

Comment:
You should change `compute[].platform.gcp`, not `controlPlane`.

And thinking about it more, the disk type should be configurable too. Some node types will require pd-balanced, other will require hyperdisk-balanced.